src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,34 @@
 package core.basesyntax;
 
 public class MainApp {
+    public MainApp(String[] machine) {
 
+        Machine truck = new Truck() {
+            @Override
+
+            public void doWork() {
```

Comment:
The overridden `doWork` method should print an informative message indicating that a `Truck` is working. Currently, it doesn't add any specific message.